samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -33,6 +41,11 @@
 
 #endif
 
+#define KiB	*1024u
+#define MiB	*1024u KiB
+#define Bytes2KiB(Bytes)	(Bytes/(1u KiB))
```

Comment:
Please use all lower-case for parameter name "bytes".

And spaces around "/" operator.

Reply:
And I'm not sure about macro name, I'd it should be either BYTES2KIB(), or just bytes2kib(), but we don't use mixed case for naming such things. (Well, perhaps it can be bytes2KiB()).

Reply:
I like last one :) Will change within next push.